pennylane/numpy/tensor.py

```diff
@@ -109,6 +109,18 @@ def __repr__(self):
         string = super().__repr__()
         return string[:-1] + ", requires_grad={})".format(self.requires_grad)
 
+    def __array_wrap__(self, obj):
+        out_arr = tensor(obj, requires_grad=self.requires_grad)
+        return super().__array_wrap__(out_arr)
+
+    def __getitem__(self, *args, **kwargs):
+        item = super().__getitem__(*args, **kwargs)
+
+        if not isinstance(item, tensor):
+            item = tensor(item, requires_grad=self.requires_grad)
+
+        return item

```

Comment:
Might be worth adding 1-2 unit test for these.

Reply:
There already are :) (in the file `test_numpy_wrapper.py`)

Reply:
Oh right, thanks! :) How come this part was separately added?

Reply:
I'm not sure why it showed up here temporarily. Perhaps a merge conflict that got resolved?